src/pricecypher/dataclasses/models.py

```diff
@@ -0,0 +1,13 @@
+from dataclasses import dataclass
+
+type Models = dict[str, Model]
```

Comment:
Type alias for a dict with keyed models. This way each model can be easily (and correctly) retrieved from the dict.

Reply:
Replaced by the [contracts/pricecypher_model.py](https://github.com/marketredesign/pricecypher_python_sdk/blob/2e1123a99f087c892ce39ae07f0e2f54934e5a93/src/pricecypher/contracts/pricecypher_model.py) class